packages/react/src/Box/Box.tsx

```diff
@@ -12,13 +12,12 @@ import type {
   TypographyProps,
 } from 'styled-system'
 import {background, border, color, flexbox, grid, layout, position, shadow, space, typography} from 'styled-system'
-import type {BetterSystemStyleObject} from '../sx'
+import type {SxProp} from '../sx'
 import sx from '../sx'
 import type {ComponentProps} from '../utils/types'
 
-type StyledBoxProps = {
-  sx?: BetterSystemStyleObject
-} & SpaceProps &
+type StyledBoxProps = SxProp &
```

Comment:
@jonrohan Any reason we would not use SxProp here?

Reply:
SxProp is deprecated, and I didn't want all Box elements to say it's deprecated since that's all the component does

Reply:
Alright, I'll revert it then